src/service_args.py

```diff
@@ -254,3 +244,16 @@ def __asset_hub_rococo(self):
         chain_json_path = f'{utils.CHAIN_SPEC_PATH}/{self.chain_name}.json'
         utils.download_chain_spec(chain_json_url, f'{self.chain_name}.json')
         self.__replace_chain_name(chain_json_path, 0)
+
+    def __bifrost(self):
+        if self.chain_name == 'bifrost':
+            chain_json_url = 'https://raw.githubusercontent.com/bifrost-finance/bifrost/develop/node/service/res/bifrost-kusama.json'
+        elif self.chain_name == 'bifrost-polkadot':
```

Comment:
@gflow33 

What do you think about renaming `'bifrost'` -> `'bifrost-kusama'`? To keep it similar to the other one. Would also better reflect the actual spec file name.

@jonathanudd Do you agree?

Reply:
I think the assumption I made is that the chain_name is gotten from the `--chain` flag in the service-args and with the current bifrost kusama parachain deployment it is `bifrost`


`
      "value": "--name=\"\\U0001F6E1 DWELLIR BIFROST KUSAMA RPC \\U0001F6E1\" --chain=bifrost --pruning=archive --ws-external --ws-port=9966 --ws-max-connections=3000 --execution=wasm --wasm-execution=compiled --rpc-external --rpc-port=9933 --rpc-cors=all --rpc-methods=Safe --prometheus-external --runtime-cache-size=10 --max-runtime-instances=40 --relay-chain-rpc-urls=ws://192.168.211.224:9933"
`
Except I have a wrong understanding of how the code works?

Me and @jonathanudd went through this when I was deploying the polkadot version and it was named `bifrost-kusama` initially, although given time constraints these changes were used in a local charm instead.

Reply:
Yes, I know that's where `chain_name` comes from and no you're not missing how it works.

My proposal is that `bifrost` changes to `bifrost-kusama` in both places, here and for the service args, to make it less confusing when using either one of the Bifrost chains. Would make us have

- `bifrost-kusama`
- `bifrost-polkadot`

and everyone would know what's what :)

Reply:
Yes I agree with you, makes it more understandable at first glance

Reply:
This means that after the next revision is released, we'll have to update the service args for the bifrost nodes when refreshing them. Let's just make sure to remember :)

Reply:
It doesn't? E.g. the client recognizes `bifrost` as `bifrost-kusama` -> no spec file is needed?

It so we should probably not have it in `service_args.py` at all :thinking:

Reply:
IIRC I was supposed to deploy a bifrost polkadot node, and then the charm only had support for the kusama node. Then we had to modify the polkadot charm to cover both instances. The kusama node has always been there and I am not really sure of the behaviour. I remember us testing out with the `--chain` and `bifrost` seemed to be the polkadot node but I am not entirely sure of this.

Reply:
Yes I will remove it